src/main/java/kg/jarkyn/server/Resource/ResourceReader.java

```diff
@@ -0,0 +1,23 @@
+package kg.jarkyn.server.Resource;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
+public class ResourceReader {
+
+    public static byte[] readFile(String filePath) {
+        try {
+            return Files.readAllBytes((new File(filePath)).toPath());
+        } catch (IOException e) {
```

Comment:
this is actually related to the fact that 404 is not yet implemented, definitely was not planning on leaving as is.
once 404 is detected further up the stack, I will have to change this
thank you for the UncheckedIOException tip :)